lib/promotion.rb

```diff
@@ -121,9 +121,9 @@ def self.recalculate(user, performed_by = nil)
     end
 
     # Then consider the group locked level
-    if user.group_locked_trust_level
+    if user.group_min_trust_level
```

Comment:
Oh my this is indeed locked today, yet the UI talks about it being minimal trust level. @nbianca we are going to have to change this whole workflow to allow for this, trust level still needs to be reviewed but after the review if it is lower than min trust level for group it should be raised.